doc/source/developing/testing.rst

```diff
@@ -287,6 +287,8 @@ Answer test examples can be found in ``yt/frontends/enzo/tests/test_outputs.py``
 How to Write Image Comparison Tests
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+TODO(#4415): update this section (maybe keep the old content but add a note for why it's out of date)
```

Comment:
I think that would be great.  Even just a `warning` callout with a link to the section below would work.

Reply:
I just realised that `generic_image` is actually not used at all anymore. I think we should just deprecate it, so I'll write the warning accordingly.